### ISSUE TITLE
Setting of learning rate and KD-Loss weight

### ISSUE THREAD

**Issue by lliai** [forjiuzhou/Spherical-Knowledge-Distillation#1]
I find that the setting of learning rate and kD-loss weight in your code is quite special. May I ask if it is consistent with your setting? Then please provide the specific operation parameters of your Resnet18/ Resnet50?
```
        if epoch < 30:
            args.alpha = 0.9
        elif epoch < 60:
            args.alpha = 0.9
        elif epoch < 80:
            args.alpha = 0.5
        elif epoch < 100:
            args.alpha = 0.1
```
```

   factor = epoch // 30
    # factor = epoch // 100
    if epoch >= 80:
        factor = factor + 1
    # if epoch >= 90:
    #     factor = factor + 1
    lr = args.lr * (0.1 ** factor)
```

**Comment by forjiuzhou** [comment]
The learning rate setting is quite common, SKD follows the setting of nvidia dali. (You can see in https://github.com/NVIDIA/DALI/blob/0db222e5574f4cc241300689e20dc22fd966a725/docs/examples/use_cases/pytorch/resnet50/main.py)

As the kD-loss weight, this is inspired by the ES ( On the Efficacy of Knowledge Distillation). Reducing the weight of kd loss could further boost the ResNet from 72.3 to 72.8 when distilled by ResNet34. Reducing the weight of kd may help to further alleviate the capacity gap problem.
